README.md

```diff
@@ -8,7 +8,7 @@ A CRDT library for JavaScript.
 
 ***Work In Progress***
 
-This module provides a set of Conflict-Free Replicated Data Types for your JavaScript programs. All CRDTs in this library, except G-Counter, are currently operation-based.
+> This module provides a set of Conflict-Free Replicated Data Types for your JavaScript programs. All CRDTs in this library, except G-Counter, are currently operation-based.
```

Comment:
The `>` is only for the text which matches the GitHub description in Standard Readme, not just the first paragraph. Here, that's "A library of Conflict-Free Replicated Data Types for JavaScript". Change that in line 7.